R/orcutt-tidiers.R

```diff
@@ -70,3 +80,26 @@ glance.orcutt <- function(x, ...) {
     nobs = stats::nobs(x)
   )
 }
+
+
+process_orcutt <- function(ret,
```

Comment:
Can we move this logic directly into `tidy.orcutt()`? I think the `process_*()` functions are unnecessary and make it harder to following the tidying logic. i.e. I would turn this PR into a single function `tidy.orcutt()` rather than three separate functions. Also, I don't think the exponentiation logic is unnecessary here -- that's typically only used for GLM/logistic type tidiers that get shoved through `tidy.lm()`.

Reply:
done!